### ISSUE TITLE
Feature request: Sync multifox settings via Firefox Sync

### ISSUE THREAD

**Issue by MurzNN** [hultmann/multifox#116]
Will be good to sync multifox settings via Firefox Sync - current mode and whitelist (and in future - blacklist) data, because now we must repeat adding each site in whitelist on each computer.

This request is not the first priority, but will be good to see in future.

**Comment by uncletoxa** [comment]
+1
that'll be awesome

**Comment by ferdynice** [comment]
Is it true that Multifox already supports this feature, though broken at times.

I swear that computer B suddenly has custom named profiles created at computer A. But, it also has an additional amount of unnamed (regularly numbered) profiles. (And somehow overwriting previously created profiles).

So in short, it seems to sync, although I prefer that it was an option I could turn off for now, because it screws up my profiles.

Thank you for the cool addon in any case.

**Comment by MurzNN** [comment]
Confirm, sync created profile list works! But sometimes profile names are reset to default "Profile 2, Profile 3" on all computers in sync account.
Seems this happens when I add new computer to sync account.
Can you fix this problem?

**Comment by ferdynice** [comment]
Yes, also an option to disable sync would be nice. Sometimes you need different profile names. (At work vs private computers for example).
